app/models/post.rb

```diff
@@ -1,2 +1,5 @@
 class Post < ApplicationRecord
+  def format_time
+    return "posted at #{self.created_at.strftime('%H:%M')} on #{self.created_at.strftime('%d/%m/%Y')}"
```

Comment:
This shouldn't be in the model - it is presentation logic. Prefer a PostPresenter object instead.